content/docs/iac/concepts/resources/providers.md

```diff
@@ -17,18 +17,21 @@ aliases:
 - /docs/concepts/resources/providers/
 ---
 
-A resource provider handles communications with a cloud service to create, read, update, and delete the resources you define in your Pulumi programs. Pulumi passes your code to a language host such as Node.js, waits to be notified of resource registrations, assembles a model of your desired state, and calls on the resource provider to produce that state. The resource provider translates those requests into API calls to the cloud service.
+A resource provider handles communications with a cloud or SaaS service to create, read, update, and delete the resources you define in your Pulumi programs.
 
-A resource provider is tied to the language that you use to write your programs. For example, if your cloud provider is AWS, the following providers are available:
+## Installing Providers
+
+You install a provider using your language's package management tool: npm in Node.js, PyPI in Python, etc. For example, if your cloud provider is AWS, the following providers are available:
```

Comment:
There's also 'package add' now for install parameterised providers. That's probably worth mentioning here.

Reply:
We need to at least mention parameterized providers here... good catch.